### PR TITLE
171 default program data

### LINKED ISSUE #171: Add default schemas and document registry for development
- [x] populate form_schema table
- [x] populate document registry
- [x] some default patients

Changes based on this:
- [x] set default Patient schema id in the patient endpoint

### PR DESCRIPTION
**Pull request by clemens-msupply**
Add some default program schemas, document_registry entries and patients

Closes #171

### CODE REVIEW

File: server/service/src/document/patient/update.rs [hunk 1]
```diff
@@ -83,16 +86,30 @@ impl From<RepositoryError> for UpdatePatientError {
...
 
-fn generate(user_id: &str, patient: &SchemaPatient, input: UpdatePatient) -> RawDocument {
-    RawDocument {
+fn generate(
+    connection: &StorageConnection,
+    user_id: &str,
+    patient: &SchemaPatient,
+    input: UpdatePatient,
+) -> Result<RawDocument, RepositoryError> {
+    let form_schema_id = match input.schema_id {
```
Comment (andreievg) [inline server/service/src/document/patient/update.rs:95]: Hmm, not quite sure what this change is for, there is something about ` set default Patient schema id in the patient endpoint` in the issue, but not sure what that means. 
Also why would schema_id be optional in input ? I thought we would validate input from F/E against json schema

Comment (clemens-msupply) [reply server/service/src/document/patient/update.rs]: Agree I made the schema_id required. I thought its easier for the FE but FE knows the schema_id anyway so no need to have it optional...

Based on these changes I got a bit sidetracked and added some schema mock data and document schema validation tests.

Comment (andreievg) [reply server/service/src/document/patient/update.rs]: thanks, was wondering, why is schema_id optional on RawDocument and ` #[serde(skip_serializing_if = "Option::is_none")]` ?

**Review by andreievg — APPROVED** [review]

File: server/service/src/document/patient/update.rs (hunk 1 above)
Comment (clemens-msupply) [reply server/service/src/document/patient/update.rs]: For the RawDocument its part of the "low level" document api and I don't want to put too many restrictions on it (at least not at this stage), e.g. could use documents without schema if needed.

For the serde macro there is no special reason for it, think I just wanted to make the object smaller if there is no schema...

**Pull request merged**